packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -120,9 +120,9 @@ export const RootScreen = () => {
           ) : showHomeStack ? (
             <Stack.Screen name='HomeStack' component={AppDrawerScreen} />
           ) : isSignUpRedesignEnabled ? (
-            <Stack.Screen name='SignOn' component={SignOnStack} />
+            <Stack.Screen name='SignOnStackNew' component={SignOnStack} />
```

Comment:
I like that you added `New` here, but I don't see a corresponding change, so this is basically disconnected here right?

Reply:
thanks, yeah this is to address the issue when working on the new sign-in where the hierarchy is "SignOn" -> "SignOn" -> "CreateEmail". so i added "stack" but then added "new" to prevent collision here. make sense?

Reply:
Oh I see, that makes sense. Also looking at this again I realized I'm dumb; disregard the "disconnected" comment. For some reason I was thinking this path needed to match the navigation links in the actual pages... Monday brain 🙈